### PR TITLE
Update account connection success message

### LINKED ISSUE #4747: Make the required UI changes to support post-onboarding of progressive accounts.
### Description

This issue will cover all changes required to support the UI states relating to an account which is partially onboarded (and then fully onboarded) following progressive onboarding. 

We need to implement showing the status of the onboarding (partial/full) along with how close the user is to their thresholds on the account overview page.

We must follow the design, but falling back to existing components for those cases where the result will be a mix between the current and new design experience.

### Relevant Links

Designs can be found on this P2 post: paJDYF-7m7-p2
Latest conversation about those changes: p1677489197405299-slack-C03PGHYA8TG

Please see the Figma file.  This issue will cover everything after the `Finish` of the onboarding.

### PR DESCRIPTION
**Pull request by ismaeldcom**
Fixes #4747

#### Changes proposed in this Pull Request
- Update connection success notice to match the new design. Using a different copy after PO and full KYC.
- Move it down, so it's rendered close to the other cards, leaving the rest of possible notices at the top.

With PO thresholds notices being handled in the two other issues, and the new "Account" card design being out of the MVP, I think this the only remaining UI change to be done. Do you agree?

#### Testing instructions
- Go to **Payments → Overview** and add `&wcpay-connection-success=1` to the URL.
- The `After Full KYC` notice should be visible.
- Edit [get_account_status_data](https://github.com/Automattic/woocommerce-payments/blob/f61304b494a35eb7d394030fb3a43fc8d3c0118d/includes/class-wc-payments-account.php#L228) to return `isEnabled => true` and `isComplete => false`, and refresh.
- The `After PO` notice should be visible.

You can also test it by onboarding a new account, but this PR only updates the UI, so no for the logic to be tested.

#### Screenshots
<table>
<tr><th>Before</th><td>Screenshot 2023-03-10 at 15 47 53</td>
<tr><th>After PO</th><td>po</td>
<tr><th>After Full KYC</th><td>full</td>
</table>

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [x] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [x] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: client/overview/index.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const OverviewPage = () => {
 		Array.isArray( tasksUnsorted ) && tasksUnsorted.sort( taskSort );
 	const queryParams = getQuery();
 
-	const showKycSuccessNotice =
+	const showConnectionSuccess =
```
Comment (oaratovskyi) [inline client/overview/index.js:53]: maybe we shouldn't show the notice when we have first transaction

**Review by oaratovskyi — APPROVED** [review]
Thanks for taking care of it!

> With PO thresholds notices being handled in the two other issues, and the new "Account" card design being out of the MVP, I think this is the only remaining UI change to be done. Do you agree? 

I agree. It could include APMs for fully onboarded accounts, but as discussed in p1678707026599569/1678704781.097859-slack-C03KTTK2YMA, not for now.

Pre-approving it! The only question remains whether to show the notice if there is a payment on the merchant's store. Maybe double-check with design or Brandon about it 🙂

**Comment by ismaeldcom** [comment]
Updated to match expected behavior after our discussion here in Slack (p1678704781097859-slack-C03KTTK2YMA). Now there are two different copies based on completed KYC (PO/Full).

**Comment by oaratovskyi** [comment]
Thanks a lot! 🚢

**Comment by ismaeldcom** [comment]
Updated again to match new design copy and style. Re-requesting review 🙂

**Comment by oaratovskyi** [comment]
Hey Ismael! The copy looks good, though I'm curious about the design match. I see that `line-height` is `28px` in the design and it's absent in the current version. I tried adding it and it looks not bad. WDYT?

Also the color of the bottom `p`, I mean `Take a moment to celebrate and look out for the first sale.` is `#757575` in design. I wonder if there's a reason for not implementing this?

p.s. Attaching a screen from the latest design to illustrate line height and bottom `p` color
image

**Comment by ismaeldcom** [comment]
You're right! I totally missed readding `line-height` after putting back a 2 lines header. And forgot about the paragraph color. Nice catch! Addressed in 8843843.

**Comment by botwoo** [comment]
### Test the build on Jurassic Ninja 

This feature is only available for A12s. Please log in into https://jurassic.ninja/specialops/ before clicking the link below:

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/4747-post-po-ui-changes) :rocket:

- Latest commit: 8843843e676cdcc6f3f79b4706f53e9a7dc96d88
- Build time: 2023-03-20 07:24:47 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Review by oaratovskyi — APPROVED** [review]
Looks great now! :shipit:

**Pull request merged**